client/state/sites/plans/test/selectors.js

```diff
@@ -201,7 +202,7 @@ describe( 'selectors', () => {
 			const discountPrice = getPlanDiscountedRawPrice( state, 77203074, 'bronze', { isMonthly: true } );
 			expect( discountPrice ).to.equal( 8.25 );
 		} );
-		it( 'should return null, if no discount is available', () => {
+		it( 'should return raw price, if no discount is available', () => {
```

Comment:
Do you think it was better before? I updated it since I think folks would expect a value to come back instead of null. It took me a little bit of digging to see why we were getting funky display values in the notice.